runtime-modules/content/src/lib.rs

```diff
@@ -1255,15 +1258,52 @@ decl_module! {
         }
 
         #[weight = 10_000_000] // TODO: adjust Weight
-        pub fn update_commitment(
+        pub fn update_channel_payouts(
             origin,
-            new_commitment: <T as frame_system::Trait>::Hash,
+            params: UpdateChannelPayoutsParameters<T>
         ) {
-            let sender = ensure_signed(origin)?;
-            ensure_authorized_to_update_commitment::<T>(&sender)?;
+            ensure_root(origin)?;
+
+            let new_min_cashout_allowed = params.min_cashout_allowed.unwrap_or_else(Self::min_cashout_allowed);
```

Comment:
Why do you correct the value instead of throwing the error?

Reply:
I don't understand the question, why should I throw an error here?

If `params.min_cashout_allowed` or `params.max_cashout_allowed` is `None` we want to leave the value of it intact in the storage.

If just one of the two is provided (ie. only `max_cashout_allowed`), we want to use the _current_ value of the other in order to check if `min_cashout_allowed <= max_cashout_allowed` would hold true after the update.

That's why I'm setting `new_min_cashout_allowed` variable to either the provided limit or the current one (and same goes for `new_max_cashout_allowed`) before running `ensure!(new_min_cashout_allowed <= new_max_cashout_allowed)`.

Note that the only purpose of `new_min_cashout_allowed` and `new_max_cashout_allowed` variables is this `ensure!` check, they aren't used for performing an actual update.

Reply:
Ok